pkg/cloud/gcp/gcp.go

```diff
@@ -128,25 +129,31 @@ func (g *gcpProvider) PrepareSubmarinerClusterEnv() error {
 			{Port: g.metricsPort, Protocol: "tcp"},
 		},
 	}
-	err := g.cloudPrepare.PrepareForSubmariner(input, g)
+	err := g.cloudPrepare.PrepareForSubmariner(input, g.reporter)
 	if err != nil {
 		return err
 	}
 
-	g.eventRecorder.Eventf("SubmarinerClusterEnvBuild", "the submariner cluster env is build on gcp")
+	g.reporter.Succeeded("the Submariner cluster environment has been set up on GCP")
```

Comment:
nit but we probably should capitalize here.  

```suggestion
	g.reporter.Succeeded("The Submariner cluster environment has been set up on GCP")
```